nixos/modules/services/audio/mpd.nix

```diff
@@ -136,67 +144,73 @@ in {
           parameter is omitted from the configuration.
         '';
       };
+
+      addBinariesToGlobalEnvironment = mkOption {
```

Comment:
Personally I'm not a fan of options that simply add binaries to the global environment mostly because it is very simple to have the user do so themselves. I'm not sure if my opinion on this is popular or not. :man_shrugging:

Reply:
I also don't think we need to have this option.